### DISCUSSION TITLE
Configurable pixi install location and cache dir

### DISCUSSION THREAD

**Discussion by imagejan** [prefix-dev/pixi#705] [Ideas]
Would it be possible to install pixi in a location different from the default `$HOME/.pixi`, e.g. by providing an environment variable `PIXI_HOME`?

In our environment, we access various linux machines where `$HOME` points to a network location with limited space. I'd rather like to specify different locations (e.g. on a network location with larger quota) for 1) the `.pixi` location and for 2) the Cache dir (something else than `~/.cache/rattler/cache`, similar to how I can specify the `PIP_CACHE_DIR` environment variable).

**Comment by ruben-arts** [comment]
These will both be in the next release :tada: 

- `PIXI_HOME` feature: https://github.com/prefix-dev/pixi/pull/627
- `PIXI_CACHE_DIR` feature: https://github.com/prefix-dev/pixi/pull/683
